frontend/console/src/layout/navigation/SearchInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { Search01Icon } from 'hugeicons-react'
+import type React from 'react'
+import { useEffect } from 'react'
+
+type SearchInputProps = {
+  onFocus: () => void
+}
+
+export const SearchInput: React.FC<SearchInputProps> = ({ onFocus }) => {
+  const shortcutText = window.navigator.userAgent.includes('Mac') ? '⌘ + K' : 'Ctrl + K'
+
+  useEffect(() => {
+    const handleKeydown = (event: KeyboardEvent) => {
+      if ((event.metaKey || event.ctrlKey) && event.key === 'k') {
+        event.preventDefault()
+        onFocus()
+      }
+    }
+
+    window.addEventListener('keydown', handleKeydown)
+
+    return () => {
+      window.removeEventListener('keydown', handleKeydown)
+    }
+  }, [onFocus])
+
+  return (
+    <div className='flex flex-1 items-center justify-center px-2 lg:ml-6 lg:justify-end'>
+      <div className='w-full max-w-lg lg:max-w-xs'>
+        <label htmlFor='search' className='sr-only'>
+          Search
+        </label>
+        <div
+          id='command-palette-search'
+          className='relative block w-full cursor-pointer rounded-md border border-gray-300 bg-white py-1.5 pl-10 pr-3 text-gray-900 ring-1 ring-inset ring-gray-300/10 placeholder:text-gray-400 sm:text-sm sm:leading-6 hover:border-indigo-500 hover:ring-indigo-500 focus-within:border-indigo-500 focus-within:ring-indigo-600'
```

Comment:
Could you update all the new components to have different colors for dark mode? It feels a bit glaring to have the big white box in the middle of an otherwise dark screen

Reply:
Totes! I wasn't 100% sure here either since it look "ok" in dark mode, but having a dedicated darker look will def be better here. Will do!

Reply:
![Screenshot 2024-09-05 at 1 34 28 PM](https://github.com/user-attachments/assets/86a22f70-9cd0-4c69-95f5-69958e66286b)
![Screenshot 2024-09-05 at 1 34 52 PM](https://github.com/user-attachments/assets/5545f37b-e375-4000-8146-191e2c03d1b9)

Reply:
The search bar here is interesting because the nav bar doesn't change so changing the seach bar in the nav bar might be a bit odd. Let's leave it for now and revisit if it continues to feel odd to us.